spec/system/admin/orders_spec.rb

```diff
@@ -583,39 +583,56 @@ def extract_pdf_content
             reader.pages.map(&:text)
           end
 
-          it "bulk prints invoices in pdf format" do
-            page.find("#listing_orders tbody tr:nth-child(1) input[name='bulk_ids[]']").click
-            page.find("#listing_orders tbody tr:nth-child(2) input[name='bulk_ids[]']").click
+          shared_examples "can bulk print invoices from 2 orders" do |legal_invoices_enabled|
+            context "with legal invoices feature #{legal_invoices_enabled}" do
+              before do
+                if legal_invoices_enabled == "enabled"
+                  Flipper.enable(:invoices)
+                else
+                  Flipper.disable(:invoices)
+                end
+              end
```

Comment:
I'm not sure what the best way is to make shared_examples work with enabling/disabling Flipper toggles, as [introduced in this PR](https://github.com/openfoodfoundation/openfoodnetwork/pull/11821).
I was thinking I could pass `feature: <empty>` as opposed to `feature: :invoices`; but it needs an argument. Maybe you have a good idea @mkllnk ?